src/components/Avatar.js

```diff
@@ -68,17 +76,22 @@ class Avatar extends PureComponent {
             return null;
         }
 
+        const isWorkspace = this.props.type === CONST.ICON_TYPE_WORKSPACE;
+        const iconSize = StyleUtils.getAvatarSize(this.props.size);
+
         const imageStyle = [
             StyleUtils.getAvatarStyle(this.props.size),
             ...this.props.imageStyles,
+            StyleUtils.getAvatarBorderRadius(this.props.size, this.props.type),
         ];
 
         const iconStyle = [
             StyleUtils.getAvatarStyle(this.props.size),
             styles.bgTransparent,
             ...this.props.imageStyles,
         ];
-        const iconSize = StyleUtils.getAvatarSize(this.props.size);
+
+        const iconFillColor = isWorkspace ? StyleUtils.getDefaultWorspaceAvatarColor(this.props.name).fill : this.props.fill;
```

Comment:
I think we should mention in the `fill` prop's documentation that it will be ignored if `this.props.type === CONST.ICON_TYPE_WORKSPACE`

Reply:
Oh good point!

Reply:
I think this was not done

Reply:
Ahh I see it was added to Icon 😕

Reply:
Oh wait, was I meant to add it elsewhere?

Reply:
Oops, I'll also add it to Avatar 🤦‍♀️ lol

Reply:
Well, Icon is just drilling it to Avatar.. so I'm not 100% sure... It makes a bit more sense to me in Avatar because that is the one currently making the decision on using it or not.